NUnitAPITests/Tests/Trello/PutBoardsTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+
+using NUnit.Framework;
+using NUnitAPITests.Config;
+using RestSharp;
+using System.IO;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+using NUnitAPITests.Client;
+using System.Dynamic;
+
+namespace NUnitAPITests.Tests.Trello
+{
+    public class PutBoardsTests
+    {
+        private List<string> ids;
+        private RestClient client;
+
+        [SetUp]
+        public void Setup()
+        {
+            ids = new List<string>();
+        }
+        [Test]
+        public void PutBoardTest()
+        {
+            
+            // Create Board
+            var request = new TrelloRequest("boards");
+            request.GetRequest().AddJsonBody("{\"name\": \"RestApiBoardEdit\"}");
+            var response = RequestManager.Post(TrelloClient.GetInstance(), request);
+
+            //parse response to json object
+            var jsonObject = JObject.Parse(response.Content);
+            ids.Add(jsonObject.SelectToken("id").ToString());
+
+            //edit Board
+            var request1 = new TrelloRequest($"boards/{ids}");
```

Comment:
This will not work, because `ids` is a list of strings. The resulting string of `$"boards/{ids}"` will be:`boards/System.Collections.Generic.List1[System.String]`. Please fix it.